candi/cloud-providers/aws/layouts/standard/base-infrastructure/main.tf

```diff
@@ -163,6 +163,9 @@ resource "aws_iam_role_policy" "node" {
       {
         "Effect": "Allow",
         "Action": [
+          %{ for policy in local.additional_role_policies }
```

Comment:
```suggestion
          %{for policy in local.additional_role_policies}
```

Reply:
These changes of terraform files are just to match `terraform fmt`. Not all existing files are correctly formatted, but if we add or change code, it's better to format it correctly.